src/formatters/stringFormatter.js

```diff
@@ -1,23 +1,16 @@
 import chalk from "chalk"
+import path from "path"
 import _ from "lodash"
-import formatter from "postcss-reporter/lib/formatter"
+import symbols from "log-symbols"
+import table, { getBorderCharacters } from "table"
+import utils from "postcss-reporter/lib/util"
 
-const minimalFormatter = formatter({
-  noIcon: true,
-  noPlugin: true,
-})
-
-export default function (results) {
-  let output = invalidOptionsFormatter(results)
-  output += deprecationsFormatter(results)
+const marginWidths = 9
 
-  return results.reduce((output, result) => {
-    output += minimalFormatter({
-      messages: result.warnings,
-      source: result.source,
-    })
-    return output
-  }, output)
+const levelColors = {
+  "info": "blue",
```

Comment:
Severity should be arbitrary now, I believe, except that our reporter treat `warning` and `error` special. So now it also treats `info` special.

@onigoetz How about another color for the arbitrary severity levels, like "advice"?